sdk/spring/spring-cloud-azure-appconfiguration-config-web/src/main/java/com/azure/spring/cloud/config/web/implementation/pullrefresh/AppConfigurationEventListener.java

```diff
@@ -1,10 +1,10 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
-package com.azure.spring.cloud.config.web.pullrefresh;
+package com.azure.spring.cloud.config.web.implementation.pullrefresh;
```

Comment:
do you think `refresh.pull` and `refresh.push` would be better as the package names?